app/javascript/sdk/IFrameHelper.js

```diff
@@ -39,6 +39,15 @@ export const IFrameHelper = {
       holderClassName += ` woot-widget--without-bubble`;
     }
     addClass(widgetHolder, holderClassName);
+
+    let closeBtn = document.createElement('button');
+    closeBtn.id = 'closeBtn';
+    closeBtn.onclick = window.$chatwoot.toggle;
```

Comment:
`window.$chatwoot.toggle()` just calls `IFrameHelper.events.toggleBubble()`. Since we're already here, maybe it makes more sense to just call that.